apps/web/components.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://ui.shadcn.com/schema.json",
+  "style": "new-york",
+  "rsc": false,
+  "tsx": true,
+  "tailwind": {
+    "config": "./tailwind.config.ts",
+    "css": "src/styles/globals.scss",
```

I think also they mean that the css should reference the tailwind config css file, which in our case is this file and not the global.scss file.

```suggestion
    "css": "src/styles/tailwind.css",
```